src/Onboarding/EulaModal.tsx

```diff
@@ -145,15 +141,13 @@ const EulaModal: FunctionComponent<EulaModalProps> = ({
                 label={t("label.close_icon")}
                 closeModal={() => setModalVisibility(false)}
               />
-              {html && (
-                <WebView
-                  style={{ flex: 1 }}
-                  source={{ html }}
-                  onShouldStartLoadWithRequest={
-                    shouldStartLoadWithRequestHandler
-                  }
-                />
-              )}
+              <WebView
+                style={{ flex: 1 }}
```

Comment:
How do we feel about moving this style to the styles group? It feels like overkill to have but one property but it might help with consistency.